DiffusionPreprocessing/scripts/DiffusionToStructural.sh

```diff
@@ -103,15 +103,26 @@ if [ ${GdcorrectionFlag} -eq 1 ]; then
     ${FSLDIR}/bin/convertwarp --rel --relout --warp1="$DataDirectory"/warped/fullWarp --postmat="$WorkingDirectory"/diff2str.mat --ref="$T1wRestoreImage"_${DiffRes} --out="$WorkingDirectory"/grad_unwarp_diff2str
     ${FSLDIR}/bin/applywarp --rel -i "$DataDirectory"/warped/data_warped -r "$T1wRestoreImage"_${DiffRes} -w "$WorkingDirectory"/grad_unwarp_diff2str --interp=spline -o "$T1wOutputDirectory"/data
 
+    #Create a mask covering voxels within the field of view for all volumes
+    ${FSLDIR}/bin/fslmaths "$DataDirectory"/warped/data_warped -abs -Tmin -bin "$DataDirectory"/warped/fov_mask
```

Comment:
Do we want to use the `-abs` option this in `fslmaths` command?  If a value is less than zero, shouldn't that be counted as zero in terms of creating the mask?

Reply:
In the output of eddy (to my understanding) the meaning of a zero or a very low and possibly negative number is different:
- zero is assigned to voxels outside of the initial FOV
- negative values might arise in voxels with a very low signal
While excluding voxels with a low signal from the diffusion MRI analysis is not a bad idea, here I am trying to adjust an existing brain mask (from the T1-weighted image) to exclude those voxels outside of the FOV for the diffusion data. So, I'm only interested in the voxels where the value is exactly zero.

On a more practical term: I first wrote it without the -abs command. The result is that many voxels with a very low signal (i.e., those outside of the brain) were also excluded from the mask. This included some voxels along the midline, which led to gaps in the nodif_brain_mask without actually improving the nodif_brain_mask in other ways (because the voxels actually outside of the brain were already excluded anyway in the original result from BET on the T1-weighted image).

Reply:
Ok.  Sounds reasonable.